docker/circleci/build.sh

```diff
@@ -5,9 +5,4 @@ set -e
 
```

Comment:
IIRC this circleci image was originally used for bundling sccache in the base container in CircleCI. This image is no longer used in the CI workflow due to speed tradeoff not working out in our favor, ie it takes more time to pull the custom image from our ECR than build without sccatch on CircleCI's default container image.  

I think we can clean up and remove this image.

Reply:
We can clean it up on separate PR?) Why mix all the changes in one?